packages/shared-ux/chrome/serverless_projects_documentation.md

```diff
@@ -0,0 +1,497 @@
+# Serverless project navigation
+
+## Introduction
+
+Welcome to the serverless project navigation documentation. Our tools help teams build customizable and flexible navigation for their serverless projects. Project navigation replaces the default Kibana navigation in serverless mode, providing a more tailored experience for each project.
+
+- [Serverless project navigation](#serverless-project-navigation)
+  - [Left Side Navigation](#left-side-navigation)
+    - [Navigation Tree Definition](#navigation-tree-definition)
+      - [Example](#example)
+      - [Navigation tree API](#navigation-tree-api)
+        - [`NavigationTreeDefinition`](#navigationtreedefinition)
+        - [`GroupDefinition`](#groupdefinition)
+        - [`RecentlyAccessedDefinition`](#recentlyaccesseddefinition)
+    - [React components](#react-components)
+      - [`unstyled`](#unstyled)
+    - [Important concepts](#important-concepts)
+      - [Deep links](#deep-links)
+      - [Cloud link](#cloud-links)
+      - [Preconfigured navigation sections](#preconfigured-navigation-sections)
+      - [Active navigation path](#active-navigation-path)
+  - [Breadcrumbs](#breadcrumbs)
+  - [Toolbar](#toolbar)
+  - [Global Search](#global-search)
+- [Testing](#testing)
+
+### Building Blocks
+
+The project navigation is composed of several key building blocks that work together to form a comprehensive navigation system. These building blocks include:
+
+1. **Left Side Navigation**: Allow users to navigate through different sections of the project.
+2. **Breadcrumbs**: A visual representation of the user's current location within the navigation hierarchy.
+3. **Toolbar**: A customizable toolbar that provides quick access to important actions and features.
```

Comment:
Can we call it "header action menu? This has been called many things: app bar, app toolbar, app menu bar... I think "header action menu" is clear to anyone listening, and it matches the naming in the code.